src/components/ui/TextArea/TextArea.tsx

```diff
@@ -22,12 +22,14 @@ export interface TextAreaProps extends AdditionalAntdTextAreaProps {
     maxRows?: number,
     className?: string,
     backgroundColor?: string,
-    onDrillDown?: () => void
+    onDrillDown?: () => void,
+    metaError?: string
```

Comment:
As in the previous scenario `<TextArea />` should rely on the parent `<Field />` component to handle errors and pass an appropriate `className`.  
One example of reasoning behind this is that there are many others components aside of `<TextArea />` which reuse `<ReadOnlyField />`:
* `<RadioButton />`
* `<NumberInput />`
* `<Dictionary />`
* `<DatePickerField />`
* `<PickListField />`
* `<InlinePickList />`